puzzles/templates/show_tags.html

```diff
@@ -33,7 +33,10 @@ <h5 class="modal-title" id="addtag-{{ puzzle.pk }}">Add tag to {{puzzle.name}}</
                     </form>
                 {% endfor %}
 
-                <button type="button" class="btn btn-info btn-sm" data-toggle="collapse" data-target="#collapseTagInput" aria-expanded="false" aria-controls="collapseTagInput"> Add New Tag.. </button>
+                <br>
+                <br>
+
+                <button type="button" class="btn btn-info btn-sm btn-block" data-toggle="collapse" data-target="#collapseTagInput" aria-expanded="false" aria-controls="collapseTagInput"> Add New Tag.. </button>
```

Comment:
Could we make the button in the modal popup a + sign like it is on the page itself?

Reply:
Saw the new button on Sushi's local deployment. Small nit: can we use "..." (three dots) or no dots at all?

Reply:
![Screen Shot 2019-12-15 at 4 07 52 AM](https://user-images.githubusercontent.com/1312469/70860508-80cb1a80-1ef0-11ea-970c-2586a0d45aee.png)
looks like this now

Reply:
Thanks. That looks good!